src/webview/js/js.js

```diff
@@ -908,6 +913,20 @@ documentBody.addEventListener('click', (e: MouseEvent) => {
   }
 });
 
+const revealMutedMessages = (target: Element) => {
+  let messageNode = getMessageNode(target);
```

Comment:
As the jsdoc says, this function is deprecated :slightly_smiling_face: It looks like `target.closest('.message-brief')` will give you what you're looking for.

Reply:
Fixed, and added a note to use that instead in the jsdoc deprecation comment :)

Reply:
Good idea, thanks :)